turbinia/jobs/volatility.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Job to execute volatility tasks."""
+
+from __future__ import unicode_literals
+
+from turbinia.evidence import VolatilityOutput
+from turbinia.jobs import interface
+from turbinia.jobs import manager
+from turbinia.workers.volatility import VolatilityTask
+
+
+class VolatilityJob(interface.TurbiniaJob):
+  """Volatility analysis job.
+    This will execute volatility against a piece of evidence.
+    """
+
+  evidence_input = [RawMemory]
```

Comment:
Can you try a full run-through locally on a memory sample? Will identify any other missing imports/etc